servers/rendering/renderer_rd/renderer_scene_render_rd.cpp

```diff
@@ -1078,6 +1078,8 @@ void RendererSceneRenderRD::render_scene(const Ref<RenderSceneBuffers> &p_render
 		scene_data.cam_transform = p_camera_data->main_transform;
 		scene_data.cam_projection = p_camera_data->main_projection;
 		scene_data.cam_orthogonal = p_camera_data->is_orthogonal;
+		// There's no shaderport for uints, and only 16 layers anyway. So drop sign bit
+		scene_data.camera_visible_layers = (int)p_camera_data->visible_layers;
```

Comment:
```suggestion
		scene_data.camera_visible_layers = p_camera_data->visible_layers;
```
Its simpler to just consistently use uints